src/components/PasswordCheck/index.tsx

```diff
@@ -31,27 +31,35 @@ type PasswordCheckProps = {
   rules: Rule[]
 }
 
-const PasswordCheck: FunctionComponent<PasswordCheckProps> = ({ rules }) => (
-  <Grid fluid gutter={0}>
-    <Row>
-      {rules.map(rule => (
-        <Col large={6} key={rule.name}>
-          <p css={styles.condition}>
-            <Icon
-              name={
-                rule.valid ? 'check-circle-outline' : 'close-circle-outline'
-              }
-              color={rule.valid ? 'success' : 'darkGrey'}
-              css={styles.icon}
-              size={20}
-            />
-            {rule.text}
-          </p>
-        </Col>
-      ))}
-    </Row>
-  </Grid>
-)
+const PasswordCheck: FunctionComponent<PasswordCheckProps> = ({ rules }) => {
+  const theme = useTheme()
+
+  return (
+    <Grid fluid gutter={0}>
+      <Row>
+        {rules.map(rule => (
+          <Col large={6} key={rule.name}>
+            <p css={styles.condition}>
+              <Icon
+                name={
+                  rule.valid ? 'check-circle-outline' : 'close-circle-outline'
+                }
+                color={
+                  rule.valid
+                    ? theme.colors.success.text
```

Comment:
```suggestion
                    ? theme.colors.success.textWeak
```

We are in a weak context (background is white), even if color is same now :)

Reply:
done

Reply:
yep, updated the neutral but not the success... sorry, done!